tests/unit/cli/test_completion.py

```
@@ -0,0 +1,482 @@
+        assert "_notebooklm" in result.output or ".bashrc" in result.output
+
+
+# ---------------------------------------------------------------------------
+# `_complete_notebooks` callback
+# ---------------------------------------------------------------------------
+
+
+class _Stub:
+    """Tiny stand-in for notebook / source / artifact rows used by completers."""
+
+    def __init__(self, id: str, title: str = ""):
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)